example/src/App.js

```diff
@@ -33,55 +34,80 @@ export default class App extends Component {
     }
   }
 
+  // getAccountToken calls the authorization server to get an access token
+  getAccountToken = async ( accountId) => {
+    let searchParams = new URLSearchParams({
+      accountId: accountId,
+    })
+
+    let options = {
+      method: 'GET',
+      mode: 'cors',
+    };
+  
+    // example auth server url
+    const url = `http://localhost:5000/token?${searchParams.toString()}`
+    let response = await fetch(url, options)
+    let responseOK = response && response.ok
+    if (responseOK) {
+      let data = await response.json()
+      if (!!data.access_token === true) {
+        this.setState({accessToken: data.access_token})
+        return data.access_token
+      } else {
+        console.warn("Unable to get token for requested operation.")
+        return false
+      }
+    }
+  }
+
   // this callback is called when an error occurs in revops-js
   onError = ({ message, http_status }) => {
-    if(http_status >= 500){
-      this.setState({error: true, errorMsg: 'Please contact support'})
+    if (http_status >= 500) {
+      this.setState({ error: true, errorMsg: 'Please contact support' })
     } else {
-      this.setState({error: true, errorMsg: message})
+      this.setState({ error: true, errorMsg: message })
     }
   }
 
   onValidationError = (errors) => {
     console.warn(errors)
-    this.setState({formDirty: true })
+    this.setState({ formDirty: true })
   }
 
   render() {
     return (
       <div style={backgroundStyles} className="revops-demo">
         <div>
-          <label>
-            Email
-            <input
-              type="text"
-              name="email"
-              onChange={
-                (e) => this.setState({
-                  email: e.target.value,
-                })
-              }
-            />
-          </label>
-          <PaymentMethod
-            publicKey="pk_sandbox_<your-sandbox-key>"
-            methods={['card', 'ach', 'plaid']}
-            account={{
-              accountId: "100000-3",
-              email: this.state.email,
+          <RevOps
+            createAccount={false}
```

Comment:
Maybe this should be broken out as `permissions={{ allowCreateAccount: false }}` so that we don't have to pollute the props list for new permissions.